components/Marker.jsx

```diff
@@ -22,16 +23,45 @@ const Marker = ({name, website, color, lat, lng}) => {
       lng={lng}
       onClick={() => setClicked(!clicked)}>
 
-      <Typography variant="body2">{name}</Typography>
+      <BusinessIcon iconType={iconType}></BusinessIcon>
+
+      {
+        clicked ? <Paper
+          layoutTransition
+          style={nameAndWebsiteStyle}
+          lat={lat + 10}
+          lng={lng + 30}>
+          <Typography variant="body2">{name}</Typography>
+        </Paper>:
+        ''
+      }
+
       {
         clicked && website ?
-          <WebsiteLink href={website} label="Website" color="secondary" /> :
+          <Paper
```

Comment:
let's have just one `Paper` component at the top level of this, with plain `div`s or `Typography` in place of the underlying sections. This will fix the modal styling

Reply:
To eliminate having different defined styles between the icon & texts?